src/launchpad/insights/android.py

```diff
@@ -0,0 +1,8 @@
+from launchpad.insights.insight import Insight, InsightsInput
+from launchpad.models.android import WebPOptimizationInsightResult
+
+
+class WebPOptimizationInsight(Insight[WebPOptimizationInsightResult]):
```

Comment:
Is this the best place to put android specific analysis?
I find it hard to navigate to the right file if everything is named `android.py` .

Reply:
moved to `android/image_optimization.py`